doc/src/docs/InputOutputReference/01d-InputOutputReference.md

```diff
@@ -4278,6 +4278,239 @@ This is the total heating output of the DX coil in Joules over the time step bei
+
+#### Cooling Coil Latent Cooling Energy [J]
+
+This is the latent cooling output of the DX coil in Joules for the time step being reported. This is determined by the inlet and outlet air conditions and the air mass flow rate through the coil.
+
+#### Cooling Coil Runtime Fraction []
+
+This is the runtime fraction of the DX coil compressor and condenser fan(s) for the time step being reported.
+
+#### Cooling Coil VRF Evaporating Temperature [C]
+
+This is the evaporating temperature of the VRF system operating at cooling mode. This value is manipulated by the VRF system considering the load conditions of all the zones it serves. It affects the cooling coil surface temperature and thus the cooling capacity of the coil.
+
+#### Cooling Coil VRF Super Heating Degrees [C]
+
+This is the super heating degrees of the VRF system operating at cooling mode. This value is manipulated by each VRF terminal unit to adjust the cooling capacity of the coil considering the load conditions of the zone. It affects the cooling coil surface temperature and thus the cooling capacity of the coil.
+
+#### Cooling Coil Condensate Volume Flow Rate [m3/s]
+
+This is the volumetric rate of water collected as condensate from the coil. This report only appears if a water storage tank is named in the input object.
+
+#### Cooling Coil Condensate Volume [m3]
+
+This is the volume of water collected as condensate from the coil. This report only appears if a water storage tank is named in the input object.
+
+
+### Coil:Heating:DX:VariableRefrigerantFlow:FluidTemperatureControl
+
+This coil object is specifically designed for the physics based VRF model applicable for Fluid Temperature Control (VRF-FluidTCtrl). It describes the performance of the indoor unit coil of the VRF system operating at heating mode. The name of this object is entered as an input to the object *ZoneHVAC:TerminalUnit:VariableRefrigerantFlow*. The outdoor unit part of the VRF system is modeled separately (refer to *AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl* object).
+
+#### Field: Name
+
+This alpha field defines a unique user-assigned name for an instance of a VRF DX heating coil. Any reference to this DX heating coil by another object will use this name. This heating coil name must be entered in the AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl object. No other system type uses this specific coil(Ref. Coil:Heating:DX:VariableRefrigerantFlow:FluidTemperatureControl).
+
+#### Field: Availability Schedule
+
+This alpha field defines the name of the schedule (ref: Schedule) that denotes whether the DX heating coil can run during a given time period. A schedule value greater than 0 (usually 1 is used) indicates that the unit can be on during the time period. A value less than or equal to 0 (usually 0 is used) denotes that the unit must be off for the time period. If this field is blank, the schedule has values of 1 for all time periods.
+
```

Comment:
If the field is blank, there is not schedule. Change to:

If this field is blank the unit is always available.